go get shows errors

When you run `go get -u github.com/kris-nova/kubicorn/...` you will get some errors related to tests:
```
# github.com/kris-nova/kubicorn/test
kris-nova/kubicorn/test/cluster.go:34:18: reconciler.Init undefined (type cloud.Reconciler has no field or method Init)
kris-nova/kubicorn/test/cluster.go:38:29: reconciler.GetExpected undefined (type cloud.Reconciler has no field or method GetExpected)
kris-nova/kubicorn/test/cluster.go:42:27: reconciler.GetActual undefined (type cloud.Reconciler has no field or method GetActual)
kris-nova/kubicorn/test/cluster.go:59:18: reconciler.Init undefined (type cloud.Reconciler has no field or method Init)
kris-nova/kubicorn/test/cluster.go:63:27: reconciler.GetActual undefined (type cloud.Reconciler has no field or method GetActual)
kris-nova/kubicorn/test/cluster.go:81:18: reconciler.Init undefined (type cloud.Reconciler has no field or method Init)
kris-nova/kubicorn/test/cluster.go:85:29: reconciler.GetExpected undefined (type cloud.Reconciler has no field or method GetExpected)
kris-nova/kubicorn/test/cluster.go:89:27: reconciler.GetActual undefined (type cloud.Reconciler has no field or method GetActual)
kris-nova/kubicorn/test/cluster.go:106:18: reconciler.Init undefined (type cloud.Reconciler has no field or method Init)
```

However, sources are obtained and `kubicorn` is installed:
```
Kubicorn version:  {"Version":"0.0.003","GitCommit":"","BuildDate":"2017-09-19 17:58:21.3670882 +0000 UTC","GOVersion":"go1.9","GOARCH":"amd64","GOOS":"linux"}
```

--- a/test/cluster.go
+++ b/test/cluster.go
@@ -31,15 +31,11 @@ func Create(testCluster *cluster.Cluster) (*cluster.Cluster, error) {
 		return nil, err
 	}
 
-	err = reconciler.Init()
+	expected, err := reconciler.Expected(testCluster)
 	if err != nil {
 		return nil, err
 	}
-	expected, err := reconciler.GetExpected()
-	if err != nil {
-		return nil, err
-	}
-	actual, err := reconciler.GetActual()
+	actual, err := reconciler.Actual(testCluster)
 	if err != nil {
 		return nil, err
 	}
@@ -56,11 +52,8 @@ func Read(testCluster *cluster.Cluster) (*cluster.Cluster, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = reconciler.Init()
-	if err != nil {
-		return nil, err
-	}
-	actual, err := reconciler.GetActual()
+
+	actual, err := reconciler.Actual(testCluster)
 	if err != nil {
 		return nil, err
 	}
@@ -78,15 +71,11 @@ func Update(testCluster *cluster.Cluster) (*cluster.Cluster, error) {
 		return nil, err
 	}
 
-	err = reconciler.Init()
+	expected, err := reconciler.Expected(testCluster)
 	if err != nil {
 		return nil, err
 	}
-	expected, err := reconciler.GetExpected()
-	if err != nil {
-		return nil, err
-	}
-	actual, err := reconciler.GetActual()
+	actual, err := reconciler.Actual(testCluster)
 	if err != nil {
 		return nil, err
 	}
@@ -103,10 +92,7 @@ func Delete(testCluster *cluster.Cluster) (*cluster.Cluster, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = reconciler.Init()
-	if err != nil {
-		return nil, err
-	}
+
 	destroyCluster, err := reconciler.Destroy()
 	if err != nil {
 		return nil, err

